src/makim/core.py

```diff
@@ -129,6 +134,50 @@ def _call_shell_app(self, cmd):
             os._exit(MakimError.SH_KEYBOARD_INTERRUPT.value)
         os.close(fd)
 
+    def _activate_conda_environment(self, env_data: dict):
+        app = env_data.get('application', '')
+        env_name = env_data.get('name', '')
+
+        if app not in ['conda', 'mamba']:
+            self._print_error(
+                f'[EE] Unsupported virtual environment application: {app}.'
+            )
+            os._exit(MakimError.SH_ERROR_RETURN_CODE.value)
+
+        sh.Command('xonsh')(
+            '-c',
+            f'{app} init xonsh',
+            _in=sys.stdin,
+            _out=sys.stdout,
+            _err=sys.stderr,
+            _bg=False,
+            _bg_exc=False,
+            _no_err=True,
+            _env=os.environ,
+            _new_session=False,
+        )
```

Comment:
Here, I try to initialize the shell.

Reply:
Just checking, also with this code, it is not working, right?

Reply:
Not sure because when I try with mamba (app), it will initialise in the shell but I don't think it performs the same with conda as mamba.

But after the initialisation of mamba in the shell, it still gives the same error.